### DISCUSSION TITLE
Surface heat flux prediction

### DISCUSSION THREAD

**Discussion by alain-coimbra** [firemodels/fds#13108] [General]
Hello everyone,

I am currently working on a project using FDS and have encountered some challenges that I would like to discuss with the community. Specifically, I have found that FDS can be largely unpredictive of the total and radiative surface heat fluxes when a flame is close to a wall.

To illustrate the issue, I have included some example figures below:

![fig1](https://github.com/firemodels/fds/assets/173062887/dac55f6a-ce70-42d4-9a00-d340ea1396a5)
Figure 1: Large-scale model of a façade fire

![fig2](https://github.com/firemodels/fds/assets/173062887/f879dd30-cf8c-4d1b-a7a3-bc71230cc974)
Figure 2: Total and Radiative Heat Flux Prediction on inert panel (with two tested values of radiative fraction for methane)
 
![fig3](https://github.com/firemodels/fds/assets/173062887/c8a9f6ee-3b14-45a5-894c-0e66e85579cf)
Figure 3: Total Heat Flux Prediction on wooden panel

In these figures, you can see that the predicted heat fluxes do not align with our experimental or expected results. This discrepancy becomes particularly noticeable when a combustible panel is used, leading to significant deviations in the predicted heat transfer. The integrated engineering pyrolysis model was used, named s-Pyro in Hodges et al. 2023.

Has anyone else encountered similar issues with heat flux predictions near walls in FDS? If so, I would greatly appreciate any insights or advice on how to address this problem. Specifically, I am interested in learning about any parameters or settings you have calibrated to achieve more accurate results.

Thank you in advance for your help and contributions!

Best regards,
Alain Coimbra

**Comment by drjfloyd** [comment]
The reported uncertainty for heat flux predictions in the current guide is ~50 % (Table 16.1).  Your results look to be within that uncertainty.

Part of that large uncertainty may be the measurement accuracy of gauges under different exposure conditions is higher than we currently use in processing validation statistics. Some recent work that is ongoing shows that radiometers used to measure flame radiant heat flux when exposed to flame likely measure too large of a flux. These are calibrated using a distance radiation source where the IR transparent window used for the gauge does not heat up.  With flame exposure, the window heats up and provides an additional source of radiant and convective heat to the gauge.

A couple of things to check in your inputs:

Make sure you have suffiicient radiation angles. Have you tried increasing the default value?

Make sure you have a reasonable fuel chemistry. If you give a set of C,H,O,N and HEAT_OF_COMBUSTION that results in strange value for EPUMO2, this can impact near field temperatures.

**Comment by johodges** [comment]
In addition to the points Jason brought up, flame spread models can be sensitive to grid resolution. Our current testing indicates the S-Pyro model is less sensitive than the kinetics models, but the predictions are still grid dependent. Best practice currently is to simulate the same scenarios with a nominal/engineering resolution then one case 2x coarser and one case 2x finer. For the nominal/engineering resolution focusing on heat transfer from a local flame, my personal starting point is dx = MIN(D*/20, 5) cm.
